### DISCUSSION TITLE
More visual distinction

### DISCUSSION THREAD

**Discussion by Spidrax** [mgmeyers/obsidian-kanban#19] [Ideas & Feature Requests]
This plugin is impressively polished, both visually and functionally. But every list looks the same - I would like to be able to choose a color for each. Ideally the color would be applied intelligently to both the list background and the cards.

Also, items marked as "complete" don't look any different than any other card. Maybe completed cards could look different than other cards.

**Comment by frvkl** [comment]
It will be great if we can pick background colors for the cards. Please make the colors configurable in settings. To be able to keep within obsidian logic, perhaps by setting a tag, we determine the card color?

**Reply by tf13** [reply]
Either a tag or yaml front matter: `Kanban-color`

I'd prefer to be able to associate tags with colors in the prefs. That way I could set a color tag (#red) or use existing tags (#urgent).

Also, maybe the option for color labeling other than full background -- other possibilities include a colored border, a single vertical or horizontal line of the color, etc.

**Reply by frvkl** [reply]
When a tag class can be added/copied to the card (.kanban-plugin__item), you could style the card however you want. I could then do something like this: 

![shot_210429_080614](https://user-images.githubusercontent.com/19724110/116508597-0111d600-a8c2-11eb-91d8-8ff0f6130572.png)
